utils/land-patterns.stanza

```diff
@@ -1322,18 +1322,6 @@ public defn make-qfn-landpattern (num-pins:Int,
                         QuadFlatNoLeads, 
                         exposed-metal-heat-feature?)
 
-; Create a QFN with no thermal pad
-public defn make-qfn-landpattern (num-pins:Int, 
-                                  pitch:Double, 
-                                  package-length:Toleranced, 
-                                  terminal-length:Toleranced, 
-                                  terminal-width:Toleranced):
-  make-quad-landpattern(num-pins,
-                        pitch,
-                        package-length, package-length, 
-                        terminal-length, terminal-width, 
-                        QuadFlatNoLeads)
-
```

Comment:
@m-hilgendorf this `make-qfn-landpattern` has the same parameters as the function below. Can delete this?

Reply:
Yes! Good catch